strapi/strapi
# Migrate EditSettingsView to ts

### What does it do?

Migration EditSettingsView page to ts

### Why is it needed?

Migration of the CM to ts

### How to test it?

Run the app as usual and check you can manage a ContentType & a Component edit view layout.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
